test/integration/simple_example/inspec.yml

```diff
@@ -54,4 +54,4 @@ attributes:
 depends:
   - name: inspec-gcp
     git: https://github.com/inspec/inspec-gcp.git
-    tag: v0.10.0
```

Comment:
maybe we should open an issue to update `git:` once new inspec-gcp is released. At the moment specifying `commit` is a bit confusing.